File: serverTCC/serverTCC/Models/Moeda.cs
using System;

namespace serverTCC.Models
{
    public class Moeda
    {
        public int Id { get; set; }
        public string Simbolo { get; set; }
        public string Nome { get; set; }
        public decimal CotacaoComercial { get; set; }
        public DateTime UltimaConsulta {get; set; }
        public string Fonte {get; set; }
    }
}

File: client/www/js/models/moeda.js
function Moeda(data) {
    "use strict";
    if (!data) data = {};

    this.id = data.id;
    this.simbolo = data.simbolo;
    this.nome = data.nome;
    this.cotacaoComercial = data.cotacaoComercial;
    this.ultimaConsulta = data.ultimaConsulta;
    this.fonte = data.fonte;
}

localEntities.register(Moeda, {
    id: {
        type: 'number',
        pk: true
    },
    simbolo: {
        type: 'string'
    },
    nome: {
        type: 'string'
    },
    cotacaoComercial: {
        type: 'number'
    },
    ultimaConsulta: {
        type: 'Date'
    },
    fonte: {
        type: 'string'
    }
});